docs/meta/taking-ownership.md

```diff
@@ -0,0 +1,48 @@
+# Taking Ownership of pypdf
+
+pypdf is currently maintained by me, Martin Thoma. I want to avoid that
+pypdf ever goes unmaintained again. This document serves as a guide to avoid
+that if I become unavailable, e.g. due to severe health issues.
+
+This currently is just an abstract scenario. I'm fine and I will likely do this
+for several more years, but I have seen how projects stand still for many years
+because of the maintainer becomming inactive.
+
+I've also followed the [GitHub Deceased User Policy](https://docs.github.com/en/site-policy/other-site-policies/github-deceased-user-policy)
+and added a [pre-designated successor](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-access-to-your-personal-repositories/maintaining-ownership-continuity-of-your-personal-accounts-repositories).
+
+## What belongs to pypdf?
+
+The resources needed for maintaining pypdf are:
+
+* PyPI: [pypdf](https://pypi.org/project/pypdf/) and [PyPDF2](https://pypi.org/project/PyPDF2/)
+* Github: [pypdf](https://github.com/py-pdf/pypdf) (the repository, not the organization)
+* ReadTheDocs: [pypdf](https://readthedocs.org/projects/pypdf/) and [PyPDF2](https://readthedocs.org/projects/pypdf2/)
+
+## When may somebody take ownership?
+
+**No activity in 180 days**: If I don't answer e-mails (info@martin-thoma.de)
```

Comment:
You don't fear being spammed on that email address?

Instead, you could for example some basic email obfuscation like this:
```html
<a href="javascript:(() => { window.location.href = atob('bWFpbHRvOmluZm9AbWFydGluLXRob21hLmRl'); })()">
```

Reply:
I'm getting spammed already as this email address is used everywhere 😅